### DISCUSSION TITLE
Your recomendation about development

### DISCUSSION THREAD

**Discussion by sluy** [pocketbase/pocketbase#150] [Polls]
Hello, im discover your project few days ago and i really love it: easy, simple, faster and uses golang <3.

I will start a new web application project with a low traffic (- 10,000 registered users and hundreds connected at a time).

I would like to start using pocketbase as a "server" but the existential doubt comes to me : Sqlite  can be a good thing over this project.
What dou you recommend? Thank you very much in advance.

**Comment by ganigeorgiev** [comment]
I voted "Yes", because PocketBase was designed initially for my other open source project - Presentator, which has more-or-less the same  load requirements as your described application.

In the past, I wouldn't consider SQLite for such type projects because of the read/write locks, but nowadays it has write-ahead-log (WAL) and it actually performs better for most of the cases when your application lives entirely on a single server.

Of course, there are lot of other factors to take in consideration, but in general if you expect your database to be less than 50 GB in size total than probably PocketBase is a good choice.

A couple of things that you need to take in consideration:
- PocketBase is a hobby community project, aka. there is no dedicated team or funding behind it, so you may find the support lacking sometime.
- PocketBase was released just recently and until we reach v1.0.0, there could be some minor breaking changes. Although I'm trying to minimize them, you have to be aware of this and read the changelogs on every release (as an example, one small breaking change is planned for the next week related to [`null` handling](https://github.com/pocketbase/pocketbase/issues/122#issuecomment-1186466209)).

**Comment by Lex-2008** [comment]
If you haven't seen it, an argument in favour of using SQLite, from SQLite developers: "Appropriate Uses For SQLite":
https://www.sqlite.org/whentouse.html

If you're planning to use pocketbase as _framework_ (i.e. write go code), instead of just as a _backend_ (i.e. download a pre-built executable and use its API), then this page might be interesting: "Many Small Queries Are Efficient In SQLite": https://www.sqlite.org/np1queryprob.html

I might be biased because I :heart: SQLite, but I don't remember seeing anyone mentioning performance issues with SQLite on Hacker News, in SQLite-related discussions: https://news.ycombinator.com/from?site=sqlite.org
